Quick Documentation doesn't work due to a NPE in Quarkus Tools

![Selection_011](https://user-images.githubusercontent.com/673386/85644575-a737f280-b697-11ea-993a-66f45c2a3ed5.png)


```
2020-06-23 18:03:39,548 [4148063]   WARN - tion.LSPGotoDeclarationHandler - java.lang.NullPointerException 
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
	at com.redhat.devtools.intellij.quarkus.lsp4ij.operations.navigation.LSPGotoDeclarationHandler.getGotoDeclarationTargets(LSPGotoDeclarationHandler.java:61)
	at com.intellij.codeInsight.navigation.action.GotoDeclarationUtil.findTargetElementsFromProviders(GotoDeclarationUtil.java:21)
	at com.intellij.codeInsight.navigation.actions.GotoDeclarationAction.findTargetElementsFromProviders(GotoDeclarationAction.java:315)
	at com.intellij.codeInsight.navigation.actions.GotoDeclarationAction.findTargetElementsNoVS(GotoDeclarationAction.java:319)
	at com.intellij.codeInsight.navigation.actions.GotoDeclarationAction.findAllTargetElements(GotoDeclarationAction.java:306)
	at com.intellij.codeInsight.navigation.actions.GotoDeclarationAction.doSelectCandidate(GotoDeclarationAction.java:117)
	at com.intellij.codeInsight.navigation.actions.GotoDeclarationAction.lambda$null$0(GotoDeclarationAction.java:78)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:884)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$underModalProgress$0(ActionUtil.java:222)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:741)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$underModalProgress$1(ActionUtil.java:223)
	at com.intellij.openapi.project.DumbService.computeWithAlternativeResolveEnabled(DumbService.java:361)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$underModalProgress$2(ActionUtil.java:224)
	at com.intellij.openapi.progress.impl.CoreProgressManager$1.run(CoreProgressManager.java:241)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:932)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:482)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$new$0(ProgressRunner.java:77)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$null$3(ProgressRunner.java:233)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:166)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:627)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:572)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:153)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:233)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NullPointerException
	at com.redhat.devtools.intellij.quarkus.lsp4ij.operations.navigation.LSPGotoDeclarationHandler.toLocation(LSPGotoDeclarationHandler.java:103)
	at com.redhat.devtools.intellij.quarkus.lsp4ij.operations.navigation.LSPGotoDeclarationHandler.toElements(LSPGotoDeclarationHandler.java:71)
	at com.redhat.devtools.intellij.quarkus.lsp4ij.operations.navigation.LSPGotoDeclarationHandler.lambda$null$1(LSPGotoDeclarationHandler.java:60)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:714)
	at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:479)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
```

--- a/src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/operations/hover/LSPTextHover.java
+++ b/src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/operations/hover/LSPTextHover.java
@@ -219,6 +219,6 @@ public class LSPTextHover extends DocumentationProviderEx {
     @Nullable
     @Override
     public PsiElement getCustomDocumentationElement(@NotNull Editor editor, @NotNull PsiFile file, @Nullable PsiElement contextElement) {
-        return file.getFirstChild();
+        return null;
     }
 }

